api.md

```diff
@@ -34,10 +36,12 @@
 * state:element( element_type, data ) - Semi-private, create an element with type and data.
 
 ### Variables
-* state.player - The name of the player.
+* state.player - The name of the current player.
+* state.multiplayer - a list of players connected to the form (on node)
 * state.param - The parameters supplied by form:show.
 * state.def - The form definition.
 * state.is_inv - Boolean which is true if this form is being shown as an inventory.
+* state.nodepos - Node position if attached to node meta
```

Comment:
It may be better to depreciate state.player (but keep present for backwards compatibility) and instead use `state.viewer` which is:

```lua
{
    type = "singleplayer",
    name = "player name"
}
```

or 

```lua
{
    type = "node",
    pos = { x = 0, y = 1, z = 2},
    players = {
        "hey",
        "yo"
    }
}
```